packages/react/src/components/Persona/test-utils.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import type { IRenderFunction } from '../../Utilities';
+import type { IPersonaProps, IPersonaSharedProps } from './index';
+
+/**
+ * function to override the default onRender callbacks
+ */
+export const wrapPersona = (
```

Comment:
this pattern was causing side effect triggered by direct module invocaion of `isConformant()`, thus `isConformant` was called for `Persona` within `PersonaCoin` test. 

Also note, these kind of patterns should always trigger a red flag. if you want something reusable to be used in various test, we should always encapsulate that in a separate module.

This was exposed by more strict node 14 types that exposed deprecated usage of `module.parent` 🫡